packages/playwright/package.json

```diff
@@ -16,6 +16,6 @@
     },
     "author": "",
     "devDependencies": {
-        "@playwright/test": "1.41.1"
+        "@playwright/test": "1.40.1"
```

Comment:
Is this intentional?

Reply:
Hmm, no!  Not sure how or why that happened.  Thanks for pointing this out.